ui/admin/app/styles/app.scss

```diff
@@ -599,59 +599,68 @@
 }
 
 .ordered-series-diagram {
+  --group-title-line-height: 2.25rem;
+
   display: flex;
-  flex-direction: row;
-  align-items: flex-end;
+  margin-bottom: 1rem;
 
-  .ordered-series-diagram-item {
-    display: grid;
-    grid-template-columns: repeat(auto-fit, 75px);
-    grid-template-rows: repeat(2, 1fr);
-    gap: sizing.rems(xs) 0;
-    grid-template-areas:
-      'icon arrow'
-      'description ...';
+  &-item {
+    display: flex;
+    margin-top: var(--group-title-line-height);
 
-    .hds-icon-tile {
-      grid-area: icon;
-      justify-self: center;
+    &-content {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      text-align: center;
     }
 
-    > span {
-      text-align: center;
-      grid-area: description;
+    &-title {
+      font-weight: var(--token-typography-font-weight-semibold);
+      margin-top: 0.5rem;
+      width: 4rem;
```

Comment:
@dvonanderson the use of a fixed width here makes the items inflexible and doesn't gracefully support longer words.  We need a solution that meets the following criteria:

- Items are flexible and do not use fixed widths.
- Items shrink to the longest border appearing in the text.